auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -32,12 +32,15 @@ class CheckPullRequest extends RequestHandler {
     final GithubService gitHub = await config.createGithubService();
     final _AutoMergeQueryResult queryResult = await _parseQueryData(pullRequest, gitHub);
     if (await shouldMergePullRequest(queryResult, slug, gitHub)) {
```

Comment:
Forgot to mention in last PR that we need to check the `autosubmit` label again before merging the PR. Merge only when the `autosubmit` label still available in the PR.

This is to exclude cases where authors removed the label for some reason after we already saved the PR to the pubsub queue.

Reply:
Thanks for reminding me this Keyong, I will check the lable before merge.